src/plugins/maps_legacy/public/map/service_settings.js

```diff
@@ -255,3 +267,17 @@ export class ServiceSettings {
     return await response.json();
   }
 }
+
+function getAttributionString(emsService) {
+  const attributions = emsService.getAttributions();
+  const attributionSnippets = attributions.map((attribution) => {
+    const anchorTag = document.createElement('a');
+    anchorTag.setAttribute('rel', 'noreferrer noopener');
+    if (attribution.url.startsWith('http://') || attribution.url.startsWith('https://')) {
+      anchorTag.setAttribute('href', attribution.url);
+    }
+    anchorTag.textContent = _.escape(attribution.label);
```

Comment:
I'm not opposed to leaving it in, but is the call to `._escape` necessary here if we're setting `textContent`? `textContent` should be able to treat HTML entities as plain text, rather than interpreting them

Reply:
I can change this. I wasn't aware of all the nuances of `.textContent`. Will strip `_.escape`.